packages/components/webpack.config.js

```diff
@@ -4,51 +4,54 @@ const BundleAnalyzerPlugin = require("webpack-bundle-analyzer")
   .BundleAnalyzerPlugin;
 const path = require("path");
 
-module.exports = {
-  mode: "development",
-  entry: path.resolve(__dirname, "./src/index.tsx"),
-  output: {
-    publicPath:
-      process.env.NODE_ENV === "production"
```

Comment:
i thought `webpack -p` would automatically set NODE_ENV==='production' in this scope... it does not